### PR TITLE
Use generator v0.20. Close Gh-35

### PR DESCRIPTION
**Pull request by robdodson**
Updated to latest `yeoman-generator` and did a ton of clean up. Could definitely use a look from @arthurvr @addyosmani @samccone

### CODE REVIEW

File: el/index.js [hunk 1]
```diff
@@ -74,70 +77,85 @@ module.exports = yeoman.generators.Base.extend({
...
     if (this.flags.docs) {
-      var elementDir = 'app/elements';
-
-      // pathToElementDir = "app/elements/x-foo"
-      var pathToElementDir = path.join(elementDir, this.elementName);
 
       // copy templates/_index.html -> app/elements/x-foo/index.html (documentation page)
-      this.template(path.join(__dirname, 'templates/_index.html'), path.join(pathToElementDir, 'index.html'));
+      this.fs.copyTpl(
+        this.templatePath('_index.html'),
```
Comment (sindresorhus) [inline el/index.js]: Why is `_index.html` prefixed with underscore? Pretty sure it doesn't need to be. Same with below `_demo`.

**Comment by sindresorhus** [comment]
// @SBoudrias

File: el/index.js (hunk 1 above)
Comment (robdodson) [inline el/index.js]: I think that's a hold over from a previous time. If I recall, some generators used to use that style. I'll update

**Comment by robdodson** [comment]
Anyone else care to take a look? I'd like to merge this soon-ish because PSK has updated to a new version. Apologies for the big PR :\

**Comment by addyosmani** [comment]
Reviewed. Overall this LG to land. 

High-level comment: there are some checks you're making for WCT usage against both the PSK base and seed-element that could be generalised (e.g rather than validating WCT and test-fixture presence in two places for `package.json`, you could do it in just one and turn it into a utility function). You could similarly do the same for the npmignore/gitignore checks.

These are minor nits however and can be looked at later. Let's land this.

**Comment by robdodson** [comment]
great thanks @addyosmani!

**Pull request closed without merging**